### PR TITLE
Improve the root readme and add contributing [PLEN-210]

### PR DESCRIPTION
**Pull request by davidd-da**

### CODE REVIEW

**Review by adamma-da — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Any type of contribution is welcome; from new features, bug fixes, documentation improvements.
+
+## How to contribute?
+
+1. Fork this repository, develop and test your changes on a new branch.
+2. Submit a pull request, the title of the PR starts with the chart name (e.g. `[canton/participant]`).
+3. If your PR corresponds to an issue, add `Fixes #XXX` to your pull request description.
+
+***NOTE***: To make the Pull Requests' (PRs) testing and merging process easier, please submit
+changes to multiple charts in separate PRs. If there is too many changes to the same helm chart,
```
Comment (miklos-da) [inline CONTRIBUTING.md:14]:
```suggestion
changes to multiple charts in separate PRs. If there are too many changes to the same helm chart,
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+find */ -name README.md | xargs sed -i -ne '/^## License$/ {p; r templates/LICENSE.md' -e ':a; n; /^##.*$/ {p; b}; ba}; p;'
+```
+
+⚠️ These amazing `sed` one-liners might break in edge cases, check the diff
+
+### PR approval and release process
+
+1. Changes are manually reviewed and tested by Digital Asset.
+1. When the PR passes all tests, the PR is merged by the reviewer(s) in the GitHub `main` branch.
+1. We will release a new Helm chart version with our CI/CD system and make it available in the repository and refresh ArtifactHub.
```
Comment (miklos-da) [inline CONTRIBUTING.md:64]:
```suggestion
1. We will release a new Helm chart version with our CI/CD system and make it available in the repository and refresh Artifact Hub.
```

**Review by miklos-da — COMMENTED** [review]
Thanks for improving the readme, @davidd-da — just added a few minor suggestions.
